recipes/zappy/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "zappy" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dd372602ff1ddb25f06eded6a0c57e0a717e7a58dc29cfffc85a06a3b02cbb67
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
```

Comment:
It looks like you need `apache_beam` which is only available on python 2.7. If you don't want this to be a hard dependency, you could comment out the test for `zappy.beam`, below.